index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var greeting = require('./lib/greeting.js');
+
+//console.log(greeting.greet('sam'));
+
+process.argv.forEach(function(arg, index) {
```

Comment:
no need to iterate through argv if you're only going to take action with a single argument. use an if to check if the argument exists, then console.log.